.github/pull_request_assignment.yml

```diff
@@ -105,61 +100,27 @@
     paths: "specification/common-types/**"
     reviewers:
       - raych1
-      - leni-msft
-      - ruowan
-      - jianyexi
-      - xiaoxuqi-ms
    
       
 - rule:
     paths: "specification/@(providerhub|sql|appplatform|deviceupdate|servicelinker|imagebuilder|throttling|secretmanagementtestsvc|containerservice|azure-kusto)/**"
     reviewers:
       - raych1
-      - xiaoxuqi-ms
       
 - rule:
     paths: "specification/voiceservices/**"
     reviewers:
       - raych1
-      - xiaoxuqi-ms
 
-- rule:
-    paths: "specification/@(financialdataservices|standbypool)/**"
```

Comment:
Can you revert this rule and add me as reviewer as it's enabled for rpsaasdev branch?